### ISSUE TITLE
VCL Modal Accessibility Audit

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#15776]
## Description
The VCL Modal in the header should be audited to ensure it meets accessibility standards.

### Screenshots
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/9ebe18f1-7238-400f-83c8-08d11c64950d)

## Acceptance Criteria
- [ ] Accessibility audit has been completed
- [ ] Recommended updates and improvements have been documented
- [ ] Any high priority/critical updates are brought to the attention of PM/DM/PO
- [ ] Design review ticket is created

## Team
Please check the team(s) that will do this work.

- [ ] CMS Team
- [x] Public Websites
- [ ] Facilities
- [ ] Accelerated Publishing

**Comment by laflannery** [comment]
Question to ask someone, but I don't know who that someone is - The VCL modal is actually [a component in the DS](https://design.va.gov/components/modal/crisis-line-modal) but we can't add this yet because it would break the injected header because that won't support components. So my question is what should I audit? Auditing production when we know we should ultimately be updating at some point in the future to the component seems like a silly thing to do

**Comment by laflannery** [comment]
Discussed in refinement - would probably be beneficial to take a look at both because while we will end up using the component in the VA header, the custom implementation will have to remain in the injected header.
